tests/fixtures/Authorization.json

```diff
@@ -0,0 +1,19 @@
+[
+{
+    "model": "registry.authorization",
+    "pk": "431934c5-7817-412f-b630-e026b7135a99",
+    "fields": {
+        "title": "Way environmental speech movie able can.",
+        "operation_max_height": 2374,
+        "operation_altitude_system": 0,
+        "airspace_type": 0,
+        "permit_to_fly_above_crowd": true,
+        "operation_area_type": 0,
+        "risk_type": 6,
+        "authorization_type": 1,
+        "end_date": "2023-03-26T00:00:00Z",
+        "created_at": "2021-03-26T20:50:45.131Z",
+        "updated_at": "2021-03-26T20:50:45.131Z"
```

Comment:
Can you look at the sample fixture and update these files?

Reply:
Done in 2ef8fa8c3fcff67b6cafd105b39529cf5107983c